content/applications/finance/sign.rst

```diff
@@ -101,71 +139,124 @@ template.
+:guilabel:`customer` and click :guilabel:`Send`.
 
-The person signing the document fills in the :guilabel:`Signature` field and clicks
-:guilabel:`Validate & Send Completed Document`. A :guilabel:`Final Validation` page pops up where to
-add their phone number. One-time codes are sent by SMS.
+The person signing the document fills in the :guilabel:`Signature` field, then :guilabel:`Sign`, and
+clicks :guilabel:`Validate & Send Completed Document`. A :guilabel:`Final Validation` page pops up
+where to add their phone number. One-time codes are sent by SMS.
 
-.. image:: sign/final-validation.png
-   :align: center
-   :alt: fill in your phone number for final validation
+.. image:: sign/sms-verification.png
+   :alt: Add a hash to your document
 
 .. note::
    - This feature is enabled by default.
    - As soon as the :guilabel:`Extra Authentication Step` applies to a role, this validation step is
      requested for any field assigned to this role.
 
+.. _sign/itsme:
+
 Itsme®
 ~~~~~~
 
-Go to :menuselection:`Sign --> Configuration --> Settings` and enable :guilabel:`Identify with
-itsme®`.
+Itsme® authentication can be used to allow signatories to provide their identity using itsme®. This
+feature is only available in **Belgium** and the **Netherlands**.
 
-Then, go to :menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra
-Authentication Step` column for the role, and select :guilabel:`Via itsme®`.
+The feature can be enabled in :guilabel:`Sign Settings` and applies automatically to the
+:guilabel:`Customer (identified with itsme®)` role. To enable it for other roles, go to
+:menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra Authentication
+Step` column for the role, and select :guilabel:`Via itsme®`.
 
-Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch the role
-to :guilabel:`customer (identified with itsme®)`, and click :guilabel:`Validate`, and
-:guilabel:`Send`.
+Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch to any
+role configured to use the feature, and click :guilabel:`Validate` and :guilabel:`Send`.
 
 .. image:: sign/itsme-identification.png
-   :align: center
    :alt: select customer identified with itsme®
 
-The person signing the document fills in the :guilabel:`Signature` field and clicks
-:guilabel:`Validate & Send Completed Document`. A :guilabel:`Final Validation` page pops up, and the
-person must authenticate with itsme®.
+Upon signing the document, the signer completes the :guilabel:`Signature` field and proceeds by
+clicking on :guilabel:`Validate & Send Completed Document`, triggering a
+:guilabel:`Final verification` page where authentication via itsme® is required.
 
-.. note::
-   This feature is only available in Belgium and the Netherlands.
+Signatory hash
+==============
+
+Each time someone signs a document, a hash - a unique ID number - is generated to ensure
```

Comment:
```suggestion
Each time someone signs a document, a hash - a unique digital signature of the operation - is generated to ensure
```

Reply:
i want to make it somewhat clear that the _content_ the person used for signing (their name, their signature, the input fields they filled in) is used to compute this signature - so any alteration will corrupt the whole 'hash chain' and be visible to the system

here's how chatgpt puts it, which is not a bad start:
> In Odoo Sign, the integrity and inalterability of documents are safeguarded using a hashing technique. At each stage of the signing process, a unique hash is created based on the document's current content and the cumulative hashes of previous steps. This includes any information added by users, like names and signatures. This method ensures that any changes made after a signature is affixed can be easily detected, maintaining the document's authenticity and security throughout its lifecycle."